catalog-rest-service/src/main/resources/json/schema/entity/tags/tagCategory.json

```diff
@@ -109,15 +98,15 @@
       "description": "Description of the tag category.",
       "type": "string"
     },
-    "version" : {
+    "version": {
       "description": "Metadata version of the entity.",
       "$ref": "../../type/entityHistory.json#/definitions/entityVersion"
     },
-    "updatedAt" : {
-      "description": "Last update time corresponding to the new version of the entity in Unix epoch time milliseconds.",
-      "$ref": "../../type/basic.json#/definitions/timestamp"
+    "updatedAt": {
```

Comment:
Do you intend to change this to dateTime type?

Reply:
see above